tjenestebuss-integrasjon/src/main/kotlin/no/nav/pensjon/brev/tjenestebuss/tjenestebussintegrasjon/TjenestebussIntegrasjonApplication.kt

```diff
@@ -7,19 +7,27 @@ import com.typesafe.config.ConfigResolveOptions
 import com.typesafe.config.ConfigValueFactory
 import io.ktor.server.engine.*
 import io.ktor.server.netty.*
+import org.slf4j.LoggerFactory
 import java.io.File
 
+private val logger = LoggerFactory.getLogger("no.nav.pensjon.brev.tjenestebuss.tjenestebussintegrasjon.TjenestebussIntegrasjonApplication")
+
 fun main() {
-    val tjenestebussIntegrasjonConfig: Config =
-        ConfigFactory.load(ConfigParseOptions.defaults(), ConfigResolveOptions.defaults().setAllowUnresolved(true))
-            .getConfig("tjenestebussintegrasjon")
-            // resolve from secrets files when running on NAIS
-            .resolveWith(getVaultSecretConfig(), ConfigResolveOptions.defaults().setAllowUnresolved(true))
-            // Resolve from secrets folder when running locally
-            .resolveLocalSecrets("azuread", "pensjonsbrev")
-    embeddedServer(Netty, port = tjenestebussIntegrasjonConfig.getInt("port"), host = "0.0.0.0") {
-        tjenestebussIntegrationApi(tjenestebussIntegrasjonConfig)
-    }.start(wait = true)
+    try {
```

Comment:
Det blir nå en forskjell mellom tjenestebuss-appen og de andre. Her logger vi utenfor EngineMain, mens i de andre logger vi på module-oppsett. Om det skal være likt så må det her være rundt `tjenestebussIntegrationApi` funksjonskallet.

Reply:
Har generelt prøvd å leggje det lengst mogleg ut, for å plukke opp mest mogleg. Tenkjer det ikkje er så farleg at det er likt, oppsettet er jo allereie litt forskjellig mellom appane.